OpenMage/magento-lts
# Flow.js updates

### Description (*)

We use `flow.js` to upload images in the backend. However, the version in this repo is from Feb 01, 2015. 

In this PR:
- Updated flow.js to [v2.14.1](https://github.com/flowjs/flow.js/releases/tag/v2.14.1)
- Removed [fusty-flow.js](https://github.com/flowjs/fusty-flow.js) -- A JavaScript library which extends Flow.js and allows to use not resumable uploads for older browsers, such as IE7, IE8 and IE9.
- Fixed an error where setting post_max_size or upload_max_filesize to 0 causes the admin dashboard to crash when uploading a file. Ref #2432
- Fixed a typo "Disalollowed" -> "Disallowed". This was done in the phtml file but I can change that pending talks in #2429



### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes OpenMage/magento-lts#2432

### Manual testing scenarios (*)

1. Pull this PR into your source code
2. Make sure to clear cache and hard refresh the edit product page
3. Try to upload some files, hopefully all should work

### Questions or comments


### Contribution checklist (*)
